### ISSUE TITLE
Calls: makecombos -> data.frame error

### ISSUE THREAD

**Issue by hanbinlu** [dphansti/mango#10]
I got error in stage 5 of using the module makecombos

**Here is the error message**
[1] "self-ligation cutoff = 18421"
[1] "grouping PETs into interactions"
[1] "---Splitting PETs by chromosome"
[1] "---Intersecting PETs with peaks"
[1] "---Building putative interaction set"
[1] "modeling PETs based on peak depth and distance"
Error in data.frame(chr1 = rep(chrom, curlength), start1 = start1[1:curlength],  : 
  arguments imply differing number of rows: 0, 1
Calls: makecombos -> data.frame
Execution halted

**Command I used**
 nohup Rscript /home/software/**mango-1.0.3**/mango.R --bowtieref /home/software/bowtie-1.1.1/genomes/hg19/hg19 --bedtoolsgenome /home/hlu/software/bedtools2/genomes/human.hg19.genome --prefix SNY_K562_RAD21 --stages 4:5 --chromexclude chrm,chrY > test &

Thank you for your help

**Comment by dphansti** [comment]
This is an issue that has been coming up frequently.  It generally occurs when a chromosome has poor sequence coverage or is so short that there are no peaks separated by distances greater than the self-ligation cutoff.  I think the problem with your command is that you did not capitalize the 'M' in chrM.  Try running stage 5 with --chromexclude chrM,chrY.  Good luck.

For other people that may read this, if you run into this problem in the future I would try using chrominclude to explicitly state which chromosomes to use instead of chromexclude. For example, 

--chrominclude chr1,chr2,chr3,chr4,chr5,chr6,chr7,chr8,chr9,chr10,chr11,chr12,chr13,chr14,chr15,chr16,chr17,chr18,chr19,chr20,chr21,chr22

**Comment by hanbinlu** [comment]
Firstly, my problem solved by explicitly include chromosomes. Thank you so much. But I think my experiences are worthwhile to update. When I capitalized the m, it has the following error:

[1] "extending peaks"
[1] "counting reads per peak"
[1] "determining self-ligation distance"
[1] "self-ligation cutoff = 18421"
[1] "grouping PETs into interactions"
[1] "---Splitting PETs by chromosome"
[1] "---Intersecting PETs with peaks"
[1] "---Building putative interaction set"
[1] "modeling PETs based on peak depth and distance"
Error in putpairs[which(putpairs$distances < maxinteractingdist & putpairs$distances >  : 
  incorrect number of dimensions

But when I tried your suggestion of explicitly state which chromosomes to include, the program works perfectly. 

Thank you.
